### ISSUE TITLE
Editor toolbar iconfonts unexpected behavior. 

### ISSUE THREAD

**Issue by ashiqurakash** [Steemia/Steemia#87]
## Expected Behavior
After writing something in the editor input. User must be able to edit the marked content by clicking on the editor toolbar icon fonts. 

## Actual Behavior
User can't edit the marked content by just clicking on the icon fonts. Because after using any icon font from the editor toolbar user need to edit the icon font code to work it properly. 

## Steps to Reproduce the Problem

  1. To reproduce this bug go to post creation page. 
  2. Type something "abc" in the editor input. 
  3. Mark a word or a sentence you want to edit. 
  4. Top of the editor input you'll see a editor toolbar. Click on any icon font you want to edit example, "iconfont icon-bold". 
  5. Note user will see the bug that the marked word or sentence didn't edit itself after clicking on the icon font of the editor toolbar. Instead of that it gives you a code example **bold** which you need to edit again to work the code in your marked word or sentence. 

## Specifications

  - Version: Steemia v0.0.4
  - Platform: Android 7.0 
  - Recorded video: https://vimeo.com/274536288

**Comment by hsyndeniz** [comment]
Hello there,

I think it's not exactly a bug, but I'm adding it to our Roadmap to fix it. Thank you :)

**Comment by ashiqurakash** [comment]
Thank you for your valuable response PO.
In every platform this editor toolbar feature works just fine. I tried it on Github, Busy and also in Evernote Web.

**Comment by jayserdny** [comment]
@ashiqurakash We understand that other platforms have this feature. However, not all platform works in the same way. Comparing against another platform is a really bad way to reproduce a bug.

**Comment by ashiqurakash** [comment]
Yes I agree. That's why I didn't mentioned it in my issue. But if this feature works like this, Steemia content editing will be more easier and comfortable. Because user won't need to delete the ** bold ** word from the middle of the stars. I find this feature more useful and time sufficient.
